pkg/reconciler/topology/partitionset/resources.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2022 The KCP Authors.
```

Comment:
2023

Reply:
and everywhere else

Reply:
I created the PR in 2022. Anyways I can change that if it matters